src/pybind/mgr/dashboard/controllers/host.py

```diff
@@ -83,6 +86,31 @@ def get_hosts(from_ceph=True, from_orchestrator=True):
     return ceph_hosts
 
 
+def get_host(hostname: str) -> Dict:
+    """
```

Comment:
Is it possible to filter results by hostname from `get_hosts` for the same functionality?

Reply:
Done